cpp/tests/strings/repeat_strings_tests.cpp

```diff
@@ -25,8 +25,10 @@
 using namespace cudf::test::iterators;
 
 namespace {
+using INT_COL = cudf::test::fixed_width_column_wrapper<int32_t>;
```

Comment:
I could not find any tests where the `repeat_times` is not an int32 column. Can you please add such test(s)?

Reply:
Thanks for reminding me about this.
Done.